features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/h2.sql

```diff
@@ -366,6 +366,19 @@ CREATE TABLE IF NOT EXISTS IDP_AUTHENTICATOR (
 INSERT INTO IDP_AUTHENTICATOR (TENANT_ID, IDP_ID, NAME) VALUES (-1234, 1, 'samlsso');
 INSERT INTO IDP_AUTHENTICATOR (TENANT_ID, IDP_ID, NAME) VALUES (-1234, 1, 'IDPProperties');
 
+CREATE TABLE IF NOT EXISTS IDP_METADATA (
+            ID INTEGER AUTO_INCREMENT,
+            IDP_ID INTEGER,
+            NAME VARCHAR(255) NOT NULL,
+            VALUE VARCHAR(255) NOT NULL,
+            DISPLAY_NAME VARCHAR(255),
+            PRIMARY KEY (ID),
+            UNIQUE (IDP_ID, NAME),
```

Comment:
better add a name for the constraint, it will make easy to drop the constraint if needed.